tailwind.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        header: ['Fjalla One', 'sans-serif'],
+        number: ['BioRhyme', 'sans-serif'],
```

Comment:
Are we using BioRhyme? I saw on figma design that it's Monserrat font.

Reply:
@jaynaxissa we are using BioRhyme font for numbers
![image](https://github.com/cherryontech/ywgc/assets/107000157/d2cebe42-e470-48c2-9cae-f9e366864241)

Reply:
I see. Can you help us with installing other required fonts too? I think it's included in your PR but correct me if I'm wrong 😃 There's a list on Figma in Desktop/mobile typography section.

Reply:
![Screenshot 2023-08-14 at 14 35 39 (2)](https://github.com/cherryontech/ywgc/assets/115336208/17e523d4-bceb-4c35-a101-d266ec1fdd45)
![Screenshot 2023-08-14 at 14 35 47 (2)](https://github.com/cherryontech/ywgc/assets/115336208/b259568e-eab2-4125-8fcc-17c4fc49d887)

Are we not using the v3?

Reply:
@jaynaxissa High-Fidelity-Wireframes its a V3

Reply:
![image](https://github.com/cherryontech/ywgc/assets/107000157/583196b1-7f3b-4394-ac8e-9f25b5dd0569)

Reply:
Yes I understand that it's v3 high-fidelity on Figma. If you look on the v3 there's more details about the fonts on there.

Reply:
At the section called "Desktop Typography"